src/YoTennis/Controllers/HomeController.cs

```diff
@@ -73,11 +73,25 @@ public static (int count, int skip) CorrectPagination(int totalCount, int count,
             {
                 var match = await _matchListService.GetMatchService(UserId, id);
                 var state = await match.GetStateAsync();
+
+                string score = "";
```

Comment:
- [ ] Create a separate function
- [ ] use `StringBuilder` instead of `string`. google how to use it, if you don't know